builder/builder-next/worker/worker_test.go

```diff
@@ -0,0 +1,94 @@
+package worker
+
+import (
+	"testing"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"gotest.tools/v3/assert"
+	is "gotest.tools/v3/assert/cmp"
+)
+
+func TestWorkerPlatforms(t *testing.T) {
```

Comment:
I'm not sure what this test is supposed to verify as all the output logic of the function that was updated comes from implementation of `platforms.Only` but these cases in here don't trigger any functionality from it. Atm it just seems to verify that items passed in `contains` are also in items passed in `platformsFn` (but not even if there is something extra).

Reply:
Oh good point. In order to trigger `platforms.Only()` to actually do something then I need to pass real `ocispec.Platform` such as `ocispec.Platform{OS: "linux", Architecture: "amd64"}`, rather than the mocked ones I am passing. However if I do that, I fear the test will become platform dependent and behave differently across platforms. Need to think how to overcome that.

Reply:
I updated the test to use real platforms, so that `platforms.Only()` does its thing. PTAL when you get a chance. Thanks.